tests/language/completion/test_mysql_naive_completion.py

```diff
@@ -0,0 +1,109 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+from unittest.mock import Mock
+from prompt_toolkit.completion import Completion
+from prompt_toolkit.document import Document
+
+from ossdbtoolsservice.language.completion.mysqlcompleter import MySQLCompleter
+
+
+class TestNaiveCompletion(unittest.TestCase):
+    """Methods for testing non-smart completion"""
+
+    def setUp(self):
+        self.completer = MySQLCompleter(smart_completion=False)
+        self.complete_event = Mock()
+
+    def test_empty_string_completion(self):
+        text = ''
+        position = 0
+        result = set(self.completer.get_completions(
+            Document(text=text, cursor_position=position),
+            self.complete_event))
+        self.assertSetEqual(result, set(map(Completion, self.completer.all_completions)))
+
+    def test_select_keyword_completion(self):
+        text = 'SEL'
+        position = len('SEL')
+        result = set(self.completer.get_completions(
+            Document(text=text, cursor_position=position),
+            self.complete_event))
+        self.assertSetEqual(result, set([Completion(text='SELECT', start_position=-3)]))
+
+    def test_function_name_completion(self):
+        text = 'SELECT MA'
```

Comment:
Does lower case and upper case matter? or will it return the same list? if yes, we should add a test for that as well to make sure that the casing of the text does not matter in completion.

Reply:
Casing matters depending on the key_casing setting we pass in. If we specify lower casing, it will return lower case completions. I can add a casing test here too.